### ISSUE TITLE
 Hidden content is being announced by screen reader when we forward swipe from the header.

### ISSUE THREAD

**Issue by naksenzh** [highcharts/highcharts#14937]
#### Expected behaviour
Screen reader should not announce any hidden content or the content that is not being displayed on the screen so that screen reader users will not get confused.

#### Actual behaviour
When we use forward swipe after the header, screen reader is announcing hidden content that is not displayed on the screen.

#### Live demo with steps to reproduce

https://user-images.githubusercontent.com/66734515/104599750-742b6f80-5678-11eb-9f60-841670c7da90.mp4

#### Product version
-
#### Affected browser(s)
Operating System: iOS 14.0.1(iPad)
Screen Reader: Voiceover

**Comment by naksenzh** [comment]
@oysteinmoseng found another issue

**Comment by oysteinmoseng** [comment]
Hi @naksenzh, this is intentional, as explained in our [documentation](https://www.highcharts.com/docs/accessibility/accessibility-module-feature-overview#screen-readers). After user testing with screen reader users we found that the additional context is helpful for understanding the chart contents.

You can customize the region by using the [accessibility.screenReaderSection.beforeChartFormat](https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormat) option. For older versions of Highcharts, there is a function option `accessibility.screenReaderSectionFormatter` that can be used. It corresponds to the current [accessibility.screenReaderSection.beforeChartFormatter](https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormatter) option.
